src/Factories/FunFunFactory.php

```diff
@@ -1011,7 +1011,13 @@ public function getTokenGenerator(): TokenGenerator {
 
 	public function newDonationConfirmationPresenter(): DonationConfirmationHtmlPresenter {
 		return new DonationConfirmationHtmlPresenter(
-			$this->getIncludeTemplate( 'Donation_Confirmation.html.twig', [ 'piwikGoals' => [ 3 ] ] )
+			$this->getIncludeTemplate(
```

Comment:
`getIncludeTemplate` is deprecated since we're no longer using content form the Wiki. The recommended way is to change the template that gets included to extend the base layout and use `getLayoutTemplate`.